### ISSUE TITLE
[Spell] Power of the Sun King, entry 36070

### ISSUE THREAD

**Issue by Sollmyr** [Looking4Group/L4G_Core#2989]
The report is about the proc rate of item Rod of the Sun King, entry **29996** and more specifically the spell Power of the Sun King, entry **36070**.

I obtained the item last night and performed testing of the proc rate. I performed two separate tests, one in a raid environment using standard raid gear and buffs, the other on a target dummy, using no offhand or special attacks the Rod.

**Test 1 – Raid Environment**
Main Hand – Rod of the Sun King (2.7 weapon speed)
Offhand – Claw of the Phoenix (1.5 weapon speed)
Weapon speed modifiers – Slice and Dice
Other procs-based mechanics – Windfury Totem, Deadly Poison, Romulo Poison Vial.

Observed data: 2659 total white hits, 684 special attacks with MH, 56 Rod procs.
Calculating total number of MH hits from total white hits:
(1/2.7)+(1/1.5) = 1.037 total attacks per second
(1/2.7) / 1.037 = 0.357 or 35.7% of all white attacks are MH attacks
0.357*2659 = 952 MH attacks
Calculating proc rate: procs / (MH white hits + MH special attacks)
56 / (952+684) = 0.034 or **3.4% proc rate** (~0.75 PPM)

Test 1 empirical data:
http://i.imgur.com/WuI7oQe.jpg
http://i.imgur.com/BOM1ZPE.jpg

**Test 2 – Target Dummy**
Main Hand – Rod of the Sun King (2.7 weapon speed)
Offhand – none
Weapon speed modifiers – none
Other procs-based mechanics – none

Observed data: 1952 MH white hits, 66 procs
66 / 1952 = 0.0338 or **3.38% proc rate** (~0.75 PPM)

Test 2 empirical data:
http://i.imgur.com/Q8ANT4S.jpg
http://i.imgur.com/7fWFsQd.jpg

Based on the tests we can conclude the following:
-	The proc rate of the weapon on Hellfire is 3.38%-3.40% or ~0.75 PPM.
-	The proc rate is consistent in both samples, indicating the proc can occur from special attacks that hit with the MH weapon – this is the correct behavior.
-	The proc rate on Hellfire is significantly lower than the reported proc rate from available retail patch 2.1 - 2.3.3 sources below.

Source 1: ElitistJerks forum
https://web.archive.org/web/20150922131855/http://forums.elitistjerks.com/topic/22887-fury-dps-spreadsheet-for-30/
Post #14
“1. What is proc rate or PPM of Rod of the Sun King?
1. About 6 PPM, no CD. I don't think it's crucial, since it's not a very powerful proc. Maybe I'll test it properly (I would now, if the PTR was up...).”
Comment in itself does not offer much data to back the claim, however the thread it is posted in features some hardcore theorycrafting about warrior dps and weapons. Nobody corrected the poster which implies his assertion is accepted by the other posters.

Source 2: WowHead comments section
http://www.wowhead.com/item=29996/rod-of-the-sun-king#comments:id=92894
“proc chance = 27% assuming 6ppm (any rogue getting this should not have much lower than 308 hit rating which means misses are not an issue for white attack procs) SnD is always up for a rogue. = 7.8 ppm on average from white attacks Throw in a SS every 4 seconds (in actuality, there should be more) = 4.05 ppm from SS totals to an average of 11.85 ppm which is about the equivalent of 2 energy per second. (1.975 per sec) The real value would be higher b/c there is more energy from the proc and combat potency that would amount to more sinister strikes and even more procs. Also, if extra hast”
“Estimated 6 ppm, effectively adding 2 energy per second. The effect is probably a percent though, further improving the valvue of haste rating to squeeze out even more hits.”
“Wrong... It must be around 25% proc rate for 6 ppm. 1 min = 60 seconds 6 ppm = 1 proc every 10 secs 10 secs / attack speed 2.7 = 3.7 attacks between each proc 100% / 3.7 atacks = 27% Sure there will be 2 or 3 sinister strikes in there some place, but even then the proc rate must be pretty high. I'd say it's somewhere in the region of 15-20% to proc” – disregarded!
“I was kind of shocked when I heard 6 ppm. I decided to go to blasted lands a few weeks ago and test for myself. After 25 minutes of beating on a servant I came out to 3.1 ppm according to procwatch.”
“I looted this mace the other night, and after testing it, I can confidently say there's no way the proc is anywhere near 27%. I'm not sure if that was a typographical error on your part, because if it was 27% the result would be far more than 6PPM. I'm yet to extensively test it on the mobs in the blasted lands, but if I was to offer an estimate, I'd put it at a 5% proc. That being said, it's a terrific mace nonetheless, and the proc is nothing short of godly.” – disregarded!
The disregarded comments are either suspicious or venture uninformed opinion.

I have chosen to disregard the 15-20% comment, because it does not seem to offer any useful data nor a basis for the 15-20% assumption. The 15-20% range is very wide for anything the player could have derived via testing, therefore it is clear the player is simply guessing the proc rate without any data.

I have chosen to disregard the 5% comment, as the poster not only does not offer any data to back his opinion, he also seems to lack basic understanding of how PPM translates into % proc rate, claiming that a 27% proc rate would result in far more than 6 PPM, which is clearly incorrect: 100/((60/6ppm)/2.7 MH weapon speed) = 27%.  Nevertheless, because the poster does not believe or understand how 27% proc rate translates into 6 ppm, he proceeds to venture an opinion on the proc rate, while specifically mentioning that _he has not yet tested the item_ himself, but intends to do so. Given the fact that he never responded again or offered more data, it is safe to assume that any test he might have performed confirmed a higher proc rate and this discouraged him from posting again lest he have to admit his own mistake.

The 4th comment suggest a proc rate of and 3.1 PPM. The poster claims this is based on a test sample.  The results this poster claims are in contradiction with the popular results, however they should not be completely disregarded.  If you assume a more conservative proc rate of 3.1 PPM, as suggested by the only valid contradictory comment, the proc rate should be approximately 14%: 100/((60/3.1ppm)/2.7 MH weapon speed) = 13.95%.

Of the 5 comments quoted, 2 agree that the proc rate should be 6 PPM or 27% proc chance. Those 2 comments are also the highest rated of all comments pertaining to this weapon. This means that the majority of readers agreed with the conclusions of these comments. Finally, it is noteworthy that the top 2 comments that claim 6ppm were both posted during patch 2.1, while all the contradictory comments, including the 3.1 PPM one, were posted in patch 2.2.3 or later. It is possible that the proc rate was reduced sometime after patch 2.1 and until patch 2.2.3 without this being mentioned in the patch notes.

In conclusion, the majority of sources found claim a proc rate of 6 PPM or 27%. The only valid, contradictory comments still put the proc rate at 14% or 3.1 PPM. The observed proc rate on Hellfire is 3.4% or 0.75 PPM.

I kindly implore the developers to look into this issue and restore the proc rate of this item to its blizzlike retail value. Whether this value will be assumed 14% or 27% I leave at their discretion, after they have reviewed the sources provided and the quality of each source. My is that the most logical assumption, given the sources, is that the proc rate was 27% in patches 2.1 and earlier and potentially reduced to 14% in patch 2.2.3 after a stealth nerf – this is the only way to explain the contradictory comment.

Regards,
Kishkumen

**Comment by AnonXS** [comment]
```
`spell_proc_event` 36070	0	0	0	4	0	0	0	10
```

**Comment by Netherfrost** [comment]
hype? pls :(

**Comment by Vittnet** [comment]
First of when you did the swings did it also count the swings of you offhand?
About 5 months ago i did 29000 swings on a target dummy (you wont be afk/logged out in combat) with only mainhand swings no offhand swings the data it gave me was 26.88% procrate on ProcMeter.

After that i heard a friend that also tried it a bit Before me that he did 40k swings and came ot the same conclution.

**Comment by Sollmyr** [comment]
@Vittnet 

Please refer to the details of the tests above. One test is performed with an offhand weapon and the number of white attacks with main hand is mathematically derived. The other test - the one on the dummy - is performed without an offhand weapon equipped.

I am not familiar with how the item may have worked 5 months ago - it wasn't in game then and couldn't be obtained or tested. The tests reflect the current state of the weapon on the live server.
